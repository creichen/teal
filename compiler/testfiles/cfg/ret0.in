fun f(x) : int = {
  return x;
}
