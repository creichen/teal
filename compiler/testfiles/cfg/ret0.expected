Entry#0 @ 1 :
  -> Access#0
VarDecl#0 @ 1 :
ReturnStmt#0 @ 2 :
  -> Exit#0
Access#0 @ 2 :
  -> ReturnStmt#0
Exit#0 @ 3 :
